proto/docs/p4runtime.md

```diff
@@ -178,9 +178,13 @@ switch(p4_type(match.field_id())) {
   case TERNARY:
     assert(match.has_ternary())
     assert(!match.ternary().value().empty() && !match.ternary().mask().empty())
+    assert(match.ternary().value() ==
+           (match.ternary().value() & match.ternary().mask()))
```

Comment:
In accordance with the conventions above, shouldn't this be:
 assert(parseInteger(match.ternary().value()) ==
￼            (parseInteger(match.ternary().value()) & parseInteger(match.ternary().mask())))